cmd/docker/builder_test.go

```diff
@@ -98,7 +98,8 @@ echo '{"SchemaVersion":"0.1.0","Vendor":"Docker Inc.","Version":"v0.6.3","ShortD
 				dockerCli.ConfigFile().Aliases = map[string]string{"builder": "buildx"}
 			}
 
-			tcmd := newDockerCommand(dockerCli)
+			ctx := context.TODO()
```

Comment:
I think we should create a `context.Background()` instance at the top of the test function and have it consumed by each function that requires a context.  

> Background returns a non-nil, empty [Context](https://pkg.go.dev/context#Context). It is never canceled, has no values, and has no deadline. It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.

Reply:
I think (but it's been a while since I wrote this) the idea here was that we may be wiring up OTEL contexts in tests at some point, so the `TODO` context is a reminder for those to be wired up (I recall we used those for find (/replace) in moby/moby).

From a "functional" perspective, they should be identical (empty context) I think?


I can have a look if we're creating this one in the right spot; I may have kept them in this spot as a reminder to set a span for each of the sub-tests, e.g. see https://github.com/moby/moby/blob/cba87125b25a637f5ab02ae28c87c71e83913c6b/integration/build/build_test.go#L24